### ISSUE TITLE
Detail view not working on inboud email

### ISSUE THREAD

**Issue by incrm-dev** [salesagility/SuiteCRM#5297]
#### Issue
We set an imap account in a crm user with 2 folders to monitor in the inbox section.
We see the emails in list view of email module, but when we click on a specific email we get a balnk screen.  In the suitecrm.log we get those messeges 

Wed Feb 21 11:52:57 2018 [28577][dd808a91-87bd-b3a1-2751-59f88cbeee94][FATAL] Exception handling in /var/www/html/crmdev/include/MVC/Controller/SugarController.php:402
Wed Feb 21 11:52:57 2018 [28577][dd808a91-87bd-b3a1-2751-59f88cbeee94][FATAL] Exception in Controller: Expected header Date to comply with RFC882 or RFC2882, but actual is ""
Wed Feb 21 11:52:57 2018 [28577][dd808a91-87bd-b3a1-2751-59f88cbeee94][FATAL] backtrace:

Investigating we discovered that the problem is probably related to the 2 folders monitored. Seems that CRM try to serch for an email (msgNo) in a wrong folder.
Removing from DB one of the folder the detail view of the email works, but we than have different problem.

Otherwise, is possible to monitor only one subfolder of the INBOX? With version 7.9.8 it works, but after upgrade to 7.10 if we select to map only a subfolders of the inbox the CRM contnue to display the emails present in the INBOX folders, not the subfolders, without possibility to chage folder

#### Expected Behavior
Clicking on the email form the listview display the detail view of the email

#### Actual Behavior
Clicking on the email from the list view lead to a blank page

#### Steps to Reproduce
1. Configure an IMAP account with two folders to monitor
2. Switch to email list view
3. Click on of of the emil present in the list view

#### Context
I can't see any email deatil

#### Your Environment

* SuiteCRM Version used: 7.10.0
* Browser name and version: Chrome 63.0.3239.132  64 bit 
* Environment name and version (e.g. MySQL, PHP 7): php 7.0.22, Mysql 5.7.19
* Operating System and version (e.g Ubuntu 16.04): CentOS

**Comment by samus-aran** [comment]
Hi @incrm-dev.
The ability to switch Inboxes have been previously raised and this PR should resolve this.
https://github.com/salesagility/SuiteCRM/pull/5277

Let us know if this helps out.

**Comment by rholighaus** [comment]
In 7.10.1 I now see this error message in the log file, resulting in a blank e-mail detailview:

Exception in Controller: Expected header Date to comply with RFC882 or RFC2882, but actual is "Fri, 23 Feb 2018 16:29:06 +0100 (CET)"

Now this Date header looks completely okay to me...

**Comment by pgorod** [comment]
@rholighaus yours seems a duplicate of #4802. Does the fix there work for you?

**Comment by rholighaus** [comment]
This fix has already been merged into 7.10.1. but if you look at the Date header generated by the MTA in question (it is Zimbra, by the way, or more likely Postfix), it still throws the exception.

I have found a fix for this but need to find out how to get it into github.

**Comment by pgorod** [comment]
@rholighaus if your change is simple, only in one file, you can do it all from GitHub:

https://help.github.com/articles/editing-files-in-another-user-s-repository/

Just make sure that you navigate to the correct branch before starting the file edit.

The branch should be `develop` if this is just for 7.10, or `hotfix-7.8.x` if it also for the other versions, like this

![image](https://user-images.githubusercontent.com/15945027/37244807-217917a0-2486-11e8-9ac5-3aaf1fb1a01b.png)

**Comment by rholighaus** [comment]
Thanks pgorod!

Have created a pull request that works with the emails that would create blank screens at my customer's site: https://github.com/rholighaus/SuiteCRM/pull/1

**Comment by pgorod** [comment]
But you didn't set the branch like I told you to, you're doing it in `master`... you can change the destination branch to become one of the hotfix branches, but if suddenly a ton of changes you didn't make get included in the PR, that's because it's not based correctly, my advice is to start again, changing the branch first like I suggested above.

Thanks for contributing!

**Comment by rholighaus** [comment]
sorry pgorod, did what you suggested, new pull request on `development` is now:

https://github.com/rholighaus/SuiteCRM/pull/2
